tests/system/Models/PaginateModelTest.php

```diff
@@ -46,7 +46,7 @@ public function testPaginatePassPerPageParameter(): void
     public function testPaginateForQueryWithGroupBy(): void
     {
         $this->createModel(ValidModel::class);
-        $this->model->groupBy('id');
+        $this->model->select('id')->groupBy('id');
```

Comment:
Why do you need `->select('id')`?

Reply:
@kenjis 

Thanks, response.

> Why do you need ->select('id')?

The reason is that it is grouped by id.
I think you will get `ERROR 1055 (42000)` with `only_full_group_by` in MySQL, for the same reason.
Similar to the above, Oracle cannot select items that are not group by when using group by.

I think it strange that you can select anything but id.
This is because it is illegal in standard SQL.

https://dev.mysql.com/doc/refman/5.6/en/group-by-handling.html

> For example, this query is illegal in standard SQL because the nonaggregated name column in the select list does not appear in the GROUP BY: